src/ape/types/__init__.py

```diff
@@ -292,6 +334,9 @@ def filter(self, event: "ContractEvent", **kwargs) -> List[ContractLog]:
                     found_events.append(log)
         return found_events
 
+    def __contains__(self, __o: object) -> bool:
+        return any(log == __o for log in self)
```

Comment:
`Any` is a better type, and gave a different name because it's easier to read
```suggestion
    def __contains__(self, val: Any) -> bool:
        return any(log == val for log in self)
```